commands/deploy_command.go

```diff
@@ -29,29 +29,30 @@ import (
 )
 
 const (
-	extDescriptorsOpt          = "e"
-    timeoutOpt                 = "t"
-	versionRuleOpt             = "version-rule"
-	noStartOpt                 = "no-start"
-	deleteServiceKeysOpt       = "delete-service-keys"
-	keepFilesOpt               = "keep-files"
-	skipOwnershipValidationOpt = "skip-ownership-validation"
-	moduleOpt                  = "m"
-	resourceOpt                = "r"
-	allModulesOpt              = "all-modules"
-	allResourcesOpt            = "all-resources"
-	strategyOpt                = "strategy"
-	skipTestingPhase           = "skip-testing-phase"
-	skipIdleStart              = "skip-idle-start"
-	startTimeoutOpt            = "apps-start-timeout"
-	stageTimeoutOpt            = "apps-stage-timeout"
-	uploadTimeoutOpt           = "apps-upload-timeout"
-	taskExecutionTimeoutOpt    = "apps-task-execution-timeout"
-	applyNamespaceAppNamesOpt        = "apply-namespace-app-names"
-	applyNamespaceServiceNamesOpt    = "apply-namespace-service-names"
-	applyNamespaceAppRoutesOpt       = "apply-namespace-app-routes"
-	applyNamespaceAsSuffix           = "apply-namespace-as-suffix"
-	maxNamespaceSize                 = 36
+	extDescriptorsOpt             = "e"
+	timeoutOpt                    = "t"
+	versionRuleOpt                = "version-rule"
+	noStartOpt                    = "no-start"
+	deleteServiceKeysOpt          = "delete-service-keys"
+	keepFilesOpt                  = "keep-files"
+	skipOwnershipValidationOpt    = "skip-ownership-validation"
+	moduleOpt                     = "m"
+	resourceOpt                   = "r"
+	allModulesOpt                 = "all-modules"
+	allResourcesOpt               = "all-resources"
+	strategyOpt                   = "strategy"
+	skipTestingPhase              = "skip-testing-phase"
+	skipIdleStart                 = "skip-idle-start"
+	startTimeoutOpt               = "apps-start-timeout"
+	stageTimeoutOpt               = "apps-stage-timeout"
+	uploadTimeoutOpt              = "apps-upload-timeout"
+	taskExecutionTimeoutOpt       = "apps-task-execution-timeout"
+	applyNamespaceAppNamesOpt     = "apply-namespace-app-names"
+	applyNamespaceServiceNamesOpt = "apply-namespace-service-names"
+	applyNamespaceAppRoutesOpt    = "apply-namespace-app-routes"
+	applyNamespaceAsSuffix        = "apply-namespace-as-suffix"
+	maxNamespaceSize              = 36
+	shouldBackupExistingApps      = "backup-existing-apps"
```

Comment:
I see that there is a pattern the names of the variables to like the parameter name and at the end to habe Opt